src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -82,6 +87,14 @@ public Measure createVersion(String id, String versionType, String username, Str
     log.info(
         "User [{}] successfully versioned measure with ID [{}]", username, savedMeasure.getId());
 
+    ResponseEntity<String> result =
+        fhirServicesClient.saveMeasureInHapiFhir(savedMeasure, accessToken);
```

Comment:
The versioned measure is already saved to MongoDB, and if the save to FHIR fails, do we need a fallback?

Reply:
@RohitKandimalla Ok so maybe not saving it in mongodb if saving in Hapi Fhir fails?

Reply:
@RohitKandimalla Ok so I changed the code to save it in Hapi Fhir first, in case of any problems, the measure won't be saved to mongo.

Reply:
We can safely ignore failed saves to HAPI for now. We don't currently have a use case to read the measure out of HAPI, so it's mostly archival at the moment.

Do log the error, though.

Reply:
Yeah that make sense to me, if not we will have a data mismatch. @jkotanchik-SB Do you have an opinion on this ?

Reply:
ooh my bad, I didn't refresh the page, so I didn't see your comment Joe.

Reply:
@RohitKandimalla @jkotanchik-SB I just reverted the code, the error log will be in madie-fhirr-service. Thanks!